index.html

```diff
@@ -1,506 +1,487 @@
+                        coding. I’d love
+                        to help with it
+                    </p>
+                    <form id="form" action="https://formspree.io/f/xqkwbzdb"
+                        method="post">
+                        <ul class="ul">
                             <li>
-                                <a
-                                    href="https://www.linkedin.com/in/m-amin-alizadeh-60a20b1b0/">
-                                    <img src="images/Linkedin icon.svg"
-                                        alt="Linkedin icon" />
-                                </a>
+                                <label for="fname" class="fw-500 fz-17 lh-20">Name:</label>
+                                <input type="text" id="fname"
+                                    class="contact__name fw-500 fz-17 lh-20"
+                                    name="contact_name"
+                                    placeholder="Enter your name" required
+                                    maxlength="30" />
                             </li>
                             <li>
-                                <a href="#">
-                                    <img src="images/hand icon.svg"
-                                        alt="Hand icon" />
-                                </a>
+                                <label for="email" class="fw-500 fz-17 lh-20">Email:</label>
+                                <input type="email" id="email"
+                                    class="contact__email fw-500 fz-17 lh-20"
+                                    placeholder="Enter your email"
+                                    name="contact_email" required />
                             </li>
                             <li>
-                                <a href="#">
-                                    <img src="images/Twitter icon.svg"
-                                        alt="Twitter icon" />
-                                </a>
+                                <label for="message" class="fw-500 fz-17 lh-20">Message:</label>
+                                <textarea id="message" class="fw-500 fz-17
+                                    lh-20" name="contact_message"
+                                    placeholder="Write your message here"
+                                    cols="30" rows="8" maxlength="500" required></textarea>
                             </li>
                             <li>
-                                <a href="#">
-                                    <img src="images/Medium icon.svg"
-                                        alt="Medium icon" />
-                                </a>
+                                <input type="submit" value="Get It Touch"
+                                    class="btn btn--submit fw-500 fz-17 lh-24" />
+                                <p class="error fw-700 fz-20 lh-28"></p>
                             </li>
                         </ul>
```

Comment:
![image](https://github.com/noelfoka/portfolio-Microverse-version/assets/60083437/cb0a23d9-b845-4a1a-b15a-30fd5ebc1c6a)
- [ ] I can't see any labels in the Figma design you used for each input. This is an extra thing that is not matching with the given design. Kindly fix this issue.